bzt 1.9.6 (new formula)

greyfenrir opened the pull request:
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

ilovezfs commented on Formula/bzt.rb:
```diff
@@ -0,0 +1,160 @@
+
+  def install
+    virtualenv_install_with_resources
+    bin.install_symlink "#{libexec}/bin/bzt" => "bzt"
```
These shouldn't be needed since we automatically link the binaries.

greyfenrir replied:
What do you mean by 'binaries'? Contents of files or their location('{libexec}/bin')?

ilovezfs replied:
Sorry, I just mean the programs bzt, jmx2yaml, soapui2yaml that you are manually symlinking. The reason that isn't happening automatically is that you have `bzt` itself as a resource block, which means that the code we have to determine what should be linked (namely everything installed into libexec/bin after the dependencies have already been installed) isn't working properly because you've already installed bzt as a dependency. Remove the resource block for bzt and the three manually symlink lines and it should work as expected.

greyfenrir commented:
Thanks, I fixed formula according your advises. But there is one fact I confused with. I've removed bzt resource block which was generated with homebrew-pypi-poet in virtualenv. In my point it's wrong practice to remove some auto generated lines. Is there way to tune poet and get correct resources list automatically?

ilovezfs commented:
@greyfenrir you'd need to submit a pull request to poet.

ilovezfs commented:
@greyfenrir Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!
